examples/fishjam-chat/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "video-chat-example",
+  "name": "@fishjam-example/fishjam-chat",
```

Comment:
Would not it be better to just name it `fishjam-chat`?
It is shorter, easier to write and looks more natural imho.

Reply:
As @mironiasty has suggested, this will now be consistent with the web SDK.